core/editor.js

```diff
@@ -172,11 +172,11 @@
 		this.on( 'instanceReady', function() {
 			if ( this.config.startupFocus ) {
 				if ( this.config.startupFocus === 'end' ) {
-					var range =  this.createRange();
+					var range = this.createRange();
 					range.selectNodeContents( this.editable() );
-					range.shrink( CKEDITOR.SHRINK_TEXT , true );
+					range.shrink( CKEDITOR.SHRINK_ELEMENT , true );
```

Comment:
`( CKEDITOR.SHRINK_ELEMENT , true );`
There should be no space between `CKEDITOR.SHRINK_ELEMENT` and comma.